[ImageOCR] move to parameter (#31810)
demisto/content: Packs/ImageOCR/ReleaseNotes/1_1_24.md

#### Integrations

##### Image OCR

> - Removed the *skip_corrupted* argument from the ***image-ocr-list-languages*** command to be an integration parameter instead.

Reviewer: ```suggestion
- Removed the *skip_corrupted* argument from the ***image-ocr-list-languages*** command, this configuration is now an integration parameter instead.
```
